addons/web/static/src/views/list/list_renderer.scss

```diff
@@ -138,6 +138,11 @@
             width: 40px;
         }
 
+        // Give minimum width to some field to avoid overflow
+        .o_priority_cell, .o_boolean_favorite_cell, .o_state_selection_cell {
+            min-width: 25px !important;
```

Comment:
I see your goal but... what's the rationale behind the "25px" exactly ?

Reply:
Note: Upto saas-16.2

Reply:
Okay, in fact, I understood (I didn't read the whole discussion). 😇